pages/installation/os/centos.rst

```diff
@@ -42,37 +42,39 @@ Additionally, run these last steps to start MongoDB during the operating system'
 Elasticsearch
 -------------
 
-Graylog 2.0.0 and higher requires Elasticsearch 2.x, so we took the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-repositories.html#_yum_dnf>`_.
+Graylog 2.3.x can be used with Elasticsearch 5.x, please follow the installation instructions from `the Elasticsearch installation guide <https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html>`_.
```

Comment:
The link to the Elasticsearch documentation should use a specific version:

https://www.elastic.co/guide/en/elasticsearch/reference/5.4/rpm.html